storybook-docs/package.json

```diff
@@ -18,24 +18,24 @@
         "@babel/core": "^7.24.9",
```

Comment:
Forgot to update babel (you did it in other package)

https://marketplace.visualstudio.com/items?itemName=pflannery.vscode-versionlens
Will help you to see if there are new version available

Reply:
yep, I installed this after our first tuple call!